packages/vega-datasets/test_vega_datasets.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from pytest_pyodide import run_in_pyodide
+
+
+@pytest.mark.xfail_browsers(firefox="slow")
+@run_in_pyodide(packages=["vega_datasets"])
```

Comment:
We can leave this be if it's way too slow, but if it's something that takes thirty or so seconds or can complete within a reasonably finite amount of time, it's probably fine to remove the xfail and increase the default timeout by decorating it with `@pytest.mark.driver_timeout(30)`